modules/core/share_limits.py

```diff
@@ -172,6 +173,8 @@ def update_share_limits_for_group(self, group_name, group_config, torrents):
         logger.separator(
             f"Updating Share Limits for [Group {group_name}] [Priority {group_config['priority']}]", space=False, border=False
         )
+        if group_config["group_upload_speed"] and group_config["limit_upload_speed"]:
+            logger.trace("Info: group_upload_speed and limit_upload_speed both specified, ignoring limit_upload_speed.")
```

Comment:
Would it make sense for `limit_upload_speed` to override any `group_upload_speed` instead rather than the other way around?

Reply:
Do you think so? I really don't have an opinion on this one, so if it seems more natural to you then I'm perfectly okay with it.

Reply:
yeah I think it makes more sense to allow users to override the `group_upload_speed` with the `limit_upload_speed`